pkg/portal/security_test.go

```diff
@@ -1,458 +0,0 @@
-package portal
```

Comment:
This test's only value was to ensure we are running 2 middlewares (Log and AAD). with the redesign and various fixes it is hard to mock it in the same way. Since we are moving to chi in the next days and that chi allows us to list the middlewares applied on routes, I removed it to avoid unnecessary work and will try to merge the PRs at the same time so we don't lose those tests